com.chartboost.mediation/Android~/chartboost-mediation-android-bridge/src/main/java/com/chartboost/mediation/unity/EventProcessor.kt

```diff
@@ -13,7 +15,35 @@ object EventProcessor {
         = eventConsumer.accept(placementName)
 
     @JvmStatic
-    fun serializeEventWithError(placementName: String, error: ChartboostMediationAdException?, eventConsumer: EventWithErrorConsumer<String, String>)
+    fun serializeLoadEvent(
+        placementName: String, loadId: String?, data: Map<String, String>?, error: ChartboostMediationError?,
+        loadConsumer: LoadEventConsumer<String, String, String, String, Double, String>) {
```

Comment:
`loadConsumer: LoadEventConsumer<String, String, String, String, Double, String>` - is there anything we can do about this so it doesn't have to be like that?

Reply:
This is how is already in place for a long time, however, we are moving away from this approach and this will be removed when the old APIs are deprecated.